packages/op-coreutils/src/op_coreutils/bigquery/write.py

```diff
@@ -209,7 +209,7 @@ class DatePart:
     date_suffix: str
 
 
-def breakout_partitioned_df(df: pl.DataFrame) -> Generator[DatePart, None, None]:
+def breakout_partitioned_df(df: pl.DataFrame, run_check=True) -> Generator[DatePart, None, None]:
```

Comment:
wondering what your thoughts are behind the flag.  I'll share mine. 

I guess it would be good to first clarify when we should and shouldn't use partitioned tables and that might help us converge. When I think of a partitioned table I see a table for which new data will only modify at most a few of the most recent partitions (close to the current date).  Older partitions never change and so should remain untouched. 

So the 10 partitions check is there to remind people that even if the data you are obtaining has information for much older dates you should not be writing all of it all over again, only updating the most recent partitions.

Reply:
Agreed, and will change accordingly